cPP_DSC.adoc

```diff
@@ -903,43 +936,43 @@ FCS_COP.1.1/SigGen:: The TSF shall perform _digital signature generation_ in acc
 |List of Standards
 
 |RSA-PKCS
-|RSASSA-PKCS1-v1_5 using {empty}[*selection*: [.underline]#SHA-256, SHA-384, SHA-512, SHA3-256, SHA3-384, SHA3-512#] 
+|RSASSA-PKCS1-v1_5 using {empty}[*selection*: [.underline]#SHA-224, SHA-256, SHA-384, SHA-512, SHA3-224, SHA3-256, SHA3-384, SHA3-512#]
```

Comment:
SHA-224 and SHA3-224 are allowed by FCS_COP.1/Hash. They have a security strength of 112 bits against collision attacks, which is actually more than the security strength of a 2048-bit RSA key according to NIST.